website/search/file_util.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+from framework.exceptions import HTTPError
+
+
+INDEXED_TYPES = [
+    'txt',
+    'md',
+    'rtf',
+    'docx',
+    'pdf',
+]
+
+
+def is_indexed(filename):
+    extension = filename.rsplit('.')[-1]
+    indexed = extension in INDEXED_TYPES
+    return indexed
+
+
+def get_content_of_file(file_):
+    url = file_.mfr_public_download_url
+    response = requests.get(url)
+    content = response.content
+    return content
+
+
+def build_file_document(name, path, addon, include_content=True):
+    file_, created = addon.find_or_create_file_guid(path)
+    parent_id = file_.node._id
+    file_content = None
```

Comment:
```python
file_content = None
if include_content:
    file_content = get_content_of_file(file_)
```

could be written as

```python
file_content = get_content_of_file(file_) if include_content else None
```

This is more concise (not necessarily a benefit!) and elimated a variable assignment.